misk-hibernate/src/main/kotlin/misk/hibernate/HibernateModule.kt

```diff
@@ -92,6 +100,7 @@ class HibernateModule @JvmOverloads constructor(
     bind<Query.Factory>().to<ReflectionQuery.Factory>()
     bind<QueryLimitsConfig>()
       .toInstance(QueryLimitsConfig(MAX_MAX_ROWS, ROW_COUNT_ERROR_LIMIT, ROW_COUNT_WARNING_LIMIT))
+    bind<HibernateExceptionLogLevelConfig>().toInstance(logLevelConfig)
```

Comment:
This will break if multiple `HibernateModule`s are installed (e.g. a service has multiple databases, or wants to have multiple connection pools for some reason)

Reply:
I think we'll need to either add support for `.annotatedWith()` in the exception mapper, or wire this up somewhere else where it's explicitly made global and bound only once

Reply:
Did some researches about guice, `guice` can tolerate duplicated binding as long as the two bindings happened in the same level and their targets are identical, so unless someone try to bind different log level to each hibernateModule it should be fine.

Reply:
That's pretty unlikely to happen imo, @adrw are you good merging this as-is?